packages/js/test/unit/browser.test.ts

```diff
@@ -165,6 +165,23 @@ describe('browser client', function () {
         })
       })
     })
+
+    it('uses the correct notifier name', function () {
+      return new Promise<void>((resolve) => {
+        client.configure({
+          apiKey: 'testing'
+        })
+
+        client.notify('testing')
+
+        setTimeout(() => {
```

Comment:
Huh, interesting trick! So using `setTimeout` forces this code to execute after `notify()` is complete, even though there's no value set for the delay? This is weird and I'm not sure I like it 😆 

But looks like the rest of the tests in here use this already, so no reason to change that with this PR!

Reply:
I copy-pasted an existing test, but here's what I remember:
- `client.notify()` makes the http request in a promise
- A promise, even if it resolves immediately, won't be executed on the same stack frame
- In order to make sure that the assertions are done after the promise execution we use setTimeout. The code inside setTimeout will be executed in a separate stack frame, after the promise is executed.

I came up with this workaround when I refactored the code from callbacks to promises, but there may be better solutions!

Reply:
In my react-native tests, I put the assertions into an `afterNotify`... which is ok but also not as obvious and readable as I would prefer. If using `notifyAsync` is an acceptable alternative, you could make the test async and `await` it. But like I mentioned, I don't think you need to make any changes in this PR since the rest of the file is also set up like this. You could add a comment to the first test that uses this trick if you feel like being extra generous to the next person who sees this file for the first time 😆

Reply:
Good point, we could use `notifyAsync` here. `notifyAsync` came after these tests 😅 .

I'll see if I can change these to notifyAsync.

Reply:
Done!